requests/LoginController.http

```diff
@@ -0,0 +1,53 @@
+### Register admin user
+POST http://localhost:8082/v1/register
+Content-Type: application/json
+
+{
+  "userName": "justinStinkt",
+  "password": "hartNachMaggi2",
+  "confirmPassword": "hartNachMaggi2",
+  "firstName": "Kevin",
+  "lastName": "Rieger",
+  "email": "kevin.rieger@gmail.com",
+  "plz": "71672",
+  "cityName": "Marbach",
+  "street": "Schwabstraße",
+  "houseNumber": "69",
+  "isAdmin": "true"
```

Comment:
Hier verstehe ich nicht ganz, wie ein admin account angelegt werden soll. Es kann ja nicht jeder einen Admin Account angelegen können... Wenn nur ein Admin einen anderen Admin Account anlegen kann, bleibt die Frage, wie der erste angelgt werden kann...  Wir können uns dazu auch später noch eine Lösung überlegen

Reply:
Bei den Rollen war ich mir auch unsicher, da ich nicht ganz sicher war wie wir das jetzt geplant hatten. Je nachdem ob man jetzt später mehrere Rollen haben kann oder nicht, müsste man das sowieso anpassen. Aber isAdmin ist jetzt einfach nur als Parameter im RequestBody gefordert, der ja sowieso im Frontend gebaut wird.

Ist dann halt später die Modellierungsfrage wo man im Frontend Admin-Accounts anlegen kann, würde da eventuell einfach einen Admin-Account in die Datenbank einschleusen und dann weiter über ne spezielle Admin-Page anlegen